Generators/include/Generators/PrimaryGenerator.h

```diff
@@ -0,0 +1,57 @@
+// Copyright CERN and copyright holders of ALICE O2. This software is
+// distributed under the terms of the GNU General Public License v3 (GPL
+// Version 3), copied verbatim in the file "COPYING".
+//
+// See https://alice-o2.web.cern.ch/ for full licensing information.
+//
+// In applying this license CERN does not waive the privileges and immunities
+// granted to it by virtue of its status as an Intergovernmental Organization
+// or submit itself to any jurisdiction.
+
+/// \author R+Preghenella - June 2017
+
+#ifndef ALICEO2_EVENTGEN_PRIMARYGENERATOR_H_
+#define ALICEO2_EVENTGEN_PRIMARYGENERATOR_H_
+
+#include "FairPrimaryGenerator.h"
+
+namespace o2
+{
+namespace eventgen
+{
+
+/*****************************************************************/
+/*****************************************************************/
+
```

Comment:
I would add here: "Custom primary generator in order to be able to initialize O2 specific stuff."

Reply:
well, it is not just to be able to initialise O2 specific stuff... this is somewhat just the first use of the custom generator. Will add a description "in order to be able to deal with O2 specific stuff"